renderers/team_renderer.py

```diff
@@ -117,6 +117,14 @@ def render_team_details(cls, handler, team, year, is_canonical):
             district_type = DistrictType.abbrevs[district_abbrev]
             district_name = DistrictType.type_names[district_type]
 
+        last_competed = None
+        participation_query = team_query.TeamParticipationQuery(team.key_name)._query_async()
```

Comment:
This should be `participation_future = team_query.TeamParticipationQuery(team.key_name).fetch_async()` and put on line 31 next to all the other queries.